main/inc/lib/javascript/ckeditor/plugins/toolbarswitch/plugin.js

```diff
@@ -34,6 +34,9 @@ function switchMe(editor, callback) {
 	var origContentCss = editor.config.contentsCss;
```

Comment:
Apparently it works. The last comment from the plugin's author says version 4.3 of the plugin should support 4.3 and above of CKEditor and, as far as I could see, it's just not marked as supported because someone forgot to mark it.